### ISSUE TITLE
vmselect crashes when accessing URL

### ISSUE THREAD

**Issue by kshcherban-lilt** [VictoriaMetrics/VictoriaMetrics#4910]
### Describe the bug

Running vmselect in cluster mode from official helm chart. Fresh installation, no data exists.
When I try to access URL over ingress: https://vmselect-ingress/select/multitenant/prometheus

vmselect component crashes with error:
```
panic: runtime error: invalid memory address or nil pointer dereference

goroutine 453 [running]:
github.com/VictoriaMetrics/VictoriaMetrics/lib/httpserver.handlerWrapper.func1()
        github.com/VictoriaMetrics/VictoriaMetrics/lib/httpserver/httpserver.go:235 +0x5b
panic({0x959c80?, 0xe9a370?})
        runtime/panic.go:920 +0x270
github.com/VictoriaMetrics/VictoriaMetrics/lib/tenantmetrics.(*CounterMap).Get(...)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/tenantmetrics/counter_map.go:38
main.selectHandler.func1()
        github.com/VictoriaMetrics/VictoriaMetrics/app/vmselect/main.go:314 +0x22
main.selectHandler(0xc00002e15c?, {0xb?, 0x9cbcab?, 0xf017c0?}, {0xbeada8, 0xc000122c80}, 0xc0001dc000, 0xc00036a4b0, 0x0)
        github.com/VictoriaMetrics/VictoriaMetrics/app/vmselect/main.go:702 +0x2d47
main.requestHandler({0xbeada8?, 0xc000122c80}, 0xc0001dc000)
        github.com/VictoriaMetrics/VictoriaMetrics/app/vmselect/main.go:297 +0xca5
github.com/VictoriaMetrics/VictoriaMetrics/lib/httpserver.handlerWrapper(0xc0002a4990, {0xbeada8?, 0xc000122c80}, 0xc0001dc000, 0xb64760)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/httpserver/httpserver.go:344 +0xd8d
github.com/VictoriaMetrics/VictoriaMetrics/lib/httpserver.gzipHandler.func1({0xbeada8?, 0xc000122c80?}, 0xc0004d4000?)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/httpserver/httpserver.go:196 +0x31
net/http.HandlerFunc.ServeHTTP(0xe9dfa0?, {0xbeada8?, 0xc000122c80?}, 0x4?)
        net/http/server.go:2136 +0x29
github.com/klauspost/compress/gzhttp.NewWrapper.func1.1({0xbea898, 0xc0004d4000}, 0x0?)
        github.com/klauspost/compress@v1.16.7/gzhttp/compress.go:486 +0x357
net/http.HandlerFunc.ServeHTTP(0xf33960?, {0xbea898?, 0xc0004d4000?}, 0xc000064b50?)
        net/http/server.go:2136 +0x29
net/http.serverHandler.ServeHTTP({0xc00036a210?}, {0xbea898?, 0xc0004d4000?}, 0x6?)
        net/http/server.go:2938 +0x8e
net/http.(*conn).serve(0xc00039c120, {0xbeb380, 0xc00036a150})
        net/http/server.go:2009 +0x5f4
created by net/http.(*Server).Serve in goroutine 87
        net/http/server.go:3086 +0x5cb
```

Here's vmselect configuration:
```
  vmselect:
    ingress:
      enabled: true
      hosts:
        - name: victoria-metrics-select-dev.example.com
          path: /select
          port: http
      tls:
      - hosts:
        - victoria-metrics-select-dev.example.com
        secretName: victoria-metrics-select-dev-example-com-tls
      annotations:
        kubernetes.io/ingress.class: "gce-internal"
        kubernetes.io/ingress.allow-http: "false"
        cert-manager.io/cluster-issuer: cert-issuer-services
        external-dns.alpha.kubernetes.io/hostname: "victoria-metrics-select-dev.example.com."
    podAnnotations:
      prometheus.io/scrape: "true"
      prometheus.io/port: "8481"
    service:
      annotations:
        cloud.google.com/neg: '{"ingress": true}'
``` 

### To Reproduce

Access vmselect over ingress at https://vmselect-ingress-host/select/multitenant/prometheus

### Version

vmselect-20230824-081515-tags-v1.93.1-cluster-0-g54a67d439c

**Comment by zekker6** [comment]
@kshcherban-lilt Thank you for the report!
This will be fixed in #4912, vmselect will properly reject requests with `multitenant` accountID since it is not supported by vmselect.
`multitenant` accountID is only supported for ingestion, see [this doc](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#url-format) for details about supported API endpoints.

**Comment by hagen1778** [comment]
@kshcherban-lilt the issue has been addressed in https://github.com/VictoriaMetrics/VictoriaMetrics/commit/f0c06b428e5c794705fb1f70a7392b3f2920f7e0 and will be included in the next release. Meanwhile, I'd recommend following recommendations from https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4910#issuecomment-1697133143

**Comment by kshcherban-lilt** [comment]
Thank you.

**Comment by valyala** [comment]
The panic when accessing `/select/multitenant/*` url at `vmselect` has been fixed since [v1.93.2](https://docs.victoriametrics.com/CHANGELOG.html). Closing the issue as fixed.

**Comment by valyala** [comment]
FYI, the issue has been fixed also at [v1.87.8 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.87.8)
